scripts/pre_leap.sh

```diff
@@ -42,3 +42,6 @@ fi
 
 # RLM-682 versions past 2.5.2 break things
 echo "python-ldap==2.5.2" >> /opt/rpc-openstack/openstack-ansible/global-requirement-pins.txt
+
+# RLM-322 set domain name to empty
+export openstack_domain=""
```

Comment:
This needs to be an Ansible variable that needs to be injected into the overrides file to be picked up properly.  Probably ping @BjoernT to find out which config file they usually modify so we can standardize on the location to insert the variable.